src/Conjure/Language/Parser.hs

```diff
@@ -2,1087 +2,1161 @@
 
 module Conjure.Language.Parser
     ( runLexerAndParser
+    , lexAndParse
```

Comment:
Remove commented out code from this file. There was some in the "functionals" file as well.